### ISSUE TITLE
Brainstorm on TurtleBot Modules

### ISSUE THREAD

**Issue by JonasPetersenFAU** [amosproj/amos2022ss03-turtlebot-fleet-management#38]
## User story
1. As a SD
2. I want to do basic research and brainstorm on modules for TurtleBot
3. So that I can later start implementing modules for the TurtleBot
## Acceptance criteria
* Get an overview on how to start implementing these modules

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Sebastian2023** [comment]
Connection Module: Establishes MQTT connection with broker using mqtt_broker and de/encode msgs using vda5050_msgs. 
Worker Module: Fulfills orders by navigating/following the virtual line provided by sick_lidar_localization.
